sequencers/modeshape-sequencer-audio/src/main/java/org/modeshape/sequencer/audio/AudioMetadata.java

```diff
@@ -0,0 +1,301 @@
+
+/**
+ * Utility for extracting metadata from audio files.
+ */
+public class AudioMetadata {
+
+    /**
+     * Return value of {@link #getFormat()} for MP3 streams. AudioMetadata can extract metadata and audio information from
+     * MP3 streams.
+     */
+    public static final int FORMAT_MP3 = 0;
+
+    /**
+     * Return value of {@link #getFormat()} for MP4 streams. AudioMetadata can extract metadata and audio information from
+     * MP4 streams.
+     */
+    public static final int FORMAT_MP4 = 1;
+
+    /**
+     * Return value of {@link #getFormat()} for Vorbis streams. AudioMetadata can extract metadata and audio information from
+     * Vorbis stream and Ogg container.
+     */
+    public static final int FORMAT_VORBIS = 2;
+
+    /**
+     * Return value of {@link #getFormat()} for FLAC streams. AudioMetadata can extract metadata and audio information from
+     * FLAC stream and Ogg container.
+     */
+    public static final int FORMAT_FLAC = 3;
+
+    /**
+     * Return value of {@link #getFormat()} for WMA streams. AudioMetadata can extract metadata and audio information from
+     * WMA stream and ASF container.
+     */
+    public static final int FORMAT_WMA = 4;
+
+    /**
+     * Return value of {@link #getFormat()} for WAVE streams. AudioMetadata can extract metadata and audio information from
+     * WAVE stream and ASF container.
+     */
+    public static final int FORMAT_WAV = 5;
+
+
+    public static final int FORMAT_UNSUPPORTED = -1;
+
+
+    /**
+     * The names of the MIME types for all supported file formats.
+     */
+    static final String[] MIME_TYPE_STRINGS = {"audio/mpeg",
+                                               "audio/mp4", "video/mp4", "video/quicktime",
+                                               "audio/vorbis", "audio/x-vorbis", "audio/ogg",
+                                               "audio/flac", "audio/x-flac",
+                                               "audio/vnd.ms-asf", "audio/x-ms-wma", "audio/x-ms-asf",
+                                               "audio/x-wav", "audio/wav", "audio/wave"};
+
+    /**
+     * The extensions of all supported file formats. The FORMAT_xyz int constants can be used as index values for
+     * this array.
+     */
+    static final String[] FORMAT_NAMES = {"mp3", "mp4", "ogg", "flac", "wma", "wav", "real"};
+
+    private int format;
+    private InputStream in;
+    private DataInput din;
+    private AudioFile audioFile;
+
+    private Long bitrate;
+    private Integer sampleRate;
+    private Integer length;
+    private String channels;
+
+    private String title;
+    private String artist;
+    private String album;
+    private String year;
+    private String comment;
+    private String track;
+    private String genre;
+
+    List<AudioMetadataArtwork> artwork;
+
+    private String mimeType;
+
+    public AudioMetadata() {
+        // disable jaudiotagger logger
+        AudioFileIO.logger.getParent().setLevel(Level.OFF);
```

Comment:
shouldn't this be controlled by the log file configuration ? Why do we want to enforce turning off logging ?